### ISSUE TITLE
Better control of system prompts

### ISSUE THREAD

**Issue by blindpk** [aaclause/basiliskLLM#98]
If I close an conversation in the app, the system prompt disappears and there is, to my knowledge, no way to clear the conversation without clearing the system prompt. I would like to either be able to clear a conversation without clearing the system prompt, or having the ability to assign a default system prompt or something similar.
Even better, but that is of course a bigger task, would be to have a list of system prompts, like with the accounts, that you could choose from, through a combo box or similar in the interface, and maybe set a default prompt for each account, while keeping the ability to have specific prompts for specific conversations.

**Comment by fastfinge** [comment]
Another way to do this would be to allow for clearing conversations without clearing the prompt, and then allow each conversation to have a user specified name. So I could have tabs like "image description", "question answering", "creative", and so on, with the different prompts for each task.

**Comment by rperez030** [comment]
I like the idea of a default system prompt, and prompt templates to choose from. we could even have the possibility of assigning global keystrokes to create new conversations with a given template. The template could also specify if the conversation should start with a screenshot, an image of the active window, an image from the clipboard or no image. That would allow us to have a keystroke for proofreading text, another one for asking questions about the active window, a meeting summarizer, etcetera. I would also include the possibility to optionally paste text from the clipboard upon launching the new conversation. In addition to global keystrokes for individual templates, we could have one that triggers a "create conversation from template" menu containing all the templates so that we can choose what to do.

The ability to clear a conversation without closing it is interesting too, but relying solely on that approach could easily lead to excessive use of tokens in my opinion.

**Comment by blindpk** [comment]
@rperez030 I like the idea of templates and assigning keystrokes to them.
Actually, the ability to save the system prompt is currently what keeps me using the old NVDA addon this was based on.

**Comment by rperez030** [comment]
The lack of a persistent system prompt is definitely the biggest issue for me as well, but there is a lot to miss from the legacy add-on.

* The possibility to navigate between messages and copy entire responses at once.
* The automatic readout of responses without moving focus (I'll try to get to this one soon).
* The possibility to render the response as html and copy the format.
I also believe this app has so much more potential, and being screen reader agnostic also means it can be used by more people.

**Comment by blindpk** [comment]
I agree about all of these points. Yes, I also feel that this has great potential.
